src/components/CCPDashboard/PatientDetailsDialog.tsx

```diff
@@ -34,6 +50,75 @@ interface PatientDetail {
   styles?: object;
 }
 
+export const PatientDetailsDialogRouting = ({
```

Comment:
minor nit - is there a reason to export this at all?

Reply:
yes, import { PatientDetailsDialogRouting } from './components/CCPDashboard/PatientDetailsDialog'; is called in the App file